Tests/Rules/ReviewUnusedParameter.tests.ps1

```diff
@@ -0,0 +1,68 @@
+Describe "ReviewUnusedParameter" {
+    BeforeAll {
+        $RuleName = 'PSReviewUnusedParameter'
+        $RuleSeverity = "Warning"
+    }
+
+    Context "When there are violations" {
+        $Cases = @(
```

Comment:
Pester doesn't guarantee variables defined in `Describe` or `Context` blocks. You should either define this in a `BeforeAll` or inline it into the `It` call (probably the better option).

Reply:
fixed in 938a283